lib/gollum-lib/sanitization.rb

```diff
@@ -1,378 +1,52 @@
-# ~*~ encoding: utf-8 ~*~
+::Loofah::HTML5::SafeList::ACCEPTABLE_PROTOCOLS.add('apt')
+
 module Gollum
-  # Encapsulate sanitization options.
-  #
-  # This class does not yet support all options of Sanitize library.
-  # See http://github.com/rgrove/sanitize/.
   class Sanitization
 
-    # Default whitelisted elements required for MathML. See https://developer.mozilla.org/en-US/docs/Web/MathML/Element
-    # For some help on generating MATHML_ELEMENTS and MATHML_ATTRS, see https://gist.github.com/dometto/52d9cb8b45d68bfc7665e5e6683b75a0
-    MATHML_ELEMENTS = [
-      'math', 'maction', 'maligngroup', 'malignmark', 'menclose',
-      'merror', 'mfenced', 'mfrac', 'mglyph', 'mi', 'mlabeledtr',
-      'mlongdiv', 'mmultiscripts', 'mn', 'mo', 'mover', 'mpadded',
-      'mphantom', 'mroot', 'mrow', 'ms', 'mscarries', 'mscarry',
-      'msgroup', 'msline', 'mspace', 'msqrt', 'msrow', 'mstack',
-      'mstyle', 'msub', 'msup', 'msubsup', 'mtable', 'mtd',
-      'mtext', 'mtr', 'munder', 'munderover', 'semantics'
-    ].freeze
-
+    @@accepted_protocols = ::Loofah::HTML5::SafeList::ACCEPTABLE_PROTOCOLS.to_a.freeze
 
-    # Default whitelisted attributes required for MathML. See https://developer.mozilla.org/en-US/docs/Web/MathML/Attribute 
-    MATHML_ATTRS = {
```

Comment:
A whole bunch of them! See [here](https://github.com/flavorjones/loofah/blob/master/lib/loofah/html5/safelist.rb#L145) I went and compared the two arrays and there are a couple of elements that don't overlap (some in ours that are not in loofah, some in loofah that are not in ours). I just assumed loofah's defaults are sane -- or maybe they have decided not to include some for safety concerns. Either way I'm happy to yield to their judgement.

Also notice that this will only affect mathml generated in the filter chain (e.g. by a rendering gem). Gollum's own support for mathml is implemented through JS: the mathml formulas are only converted to HTML after the page is loaded, and so loofah won't break any of that functionality.